client/app/dedicated/server/bandwidth-vrack/order-agora/dedicated-server-bandwidth-agora-vrack.html

```diff
@@ -0,0 +1,58 @@
+<div data-ng-controller="ServerOrderAgoraBandwidthVrackCtrl as ctrl">
+    <div data-wizard
```

Comment:
Should we continue (& wants) to use the `wizard` ?

Reply:
No

Reply:
I cannot use a `stepper` either, will change the design of only this model. Are there any other alternatives to this ?